### PR TITLE
Feat text utilities

### PR DESCRIPTION
**Pull request by ankit9015**
Feature text utilities contains:
1. Headings
2. Different text size classes
3. Different font weight classes
4. Gray Text
5. Center Text
text utilities

### CODE REVIEW

File: components/text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.h5, 
+.h6 {
+    margin: 0.6rem 0;
+    line-height: 1.2;
+    font-weight: 500;
+}
+
+
+.h1,
+.text-xxl {
```
Comment (Avin008) [inline components/text-utilities/text-utilities.css:17]: why use both class name && element name for selection?

**Review by Avin008 — COMMENTED** [review]
The Code looks awesome. good to merge

File: components/text-utilities/text-utilities.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+.h1, 
+.h2, 
+.h3, 
+.h4, 
+.h5, 
+.h6 {
+    margin: 0.6rem 0;
+    line-height: 1.2;
+    font-weight: 500;
```
Comment (rkapoor10) [inline components/text-utilities/text-utilities.css:12]: recheck if you want the same font weight for all

**Review by rkapoor10 — APPROVED** [review]
The code is well structured. Looks good to merge

File: components/text-utilities/text-utilities.css (hunk 1 above)
Comment (ankit9015) [reply components/text-utilities/text-utilities.css:17]: I was adding style to selector as well as class, but that is not  a good strategy

**Pull request merged**